src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp

```diff
@@ -194,6 +195,9 @@ const std::map<utils::ActivationTypes, std::vector<std::vector<float>>>& activat
         {Exp,         {{}}},
         {Clamp,       {{-2.0f, 2.0f}}},
         {Elu,         {{0.1f}}},
+#if !defined(OPENVINO_ARCH_ARM)
```

Comment:
Why it is added for non ARM platforms only?

Reply:
was added for any platform except ARM32

Reply:
done, thanks